components/console-backend-service/Makefile

```diff
@@ -1,19 +1,57 @@
+ENTRYPOINT := ./main.go
```

Comment:
Could you also add `format` target? It is a simple tool to quickly run imports and fmt